.github/workflows/trunk.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           cache-dependency-path: yarn.lock
 
       - run: yarn install --frozen-lockfile
-
+      - run: yarn workspaces run prepack
```

Comment:
this shouldn't be necessary. `lerna publish` will pack along the way. I suppose you had it here for debugging? please either remove or add a comment explaining why this redundant command is included

Reply:
This was to be safe because the `lerna publish` wasn't prepacking in the correct order and failing as seen in https://github.com/Agoric/ui-kit/actions/runs/8053306725/job/21995398192

I read that lerna would pack in the correct order if we explicitly added the dependencies in the `package.jsons`, but added this to be safe because I noticed [MAINTAINERS.md](https://github.com/Agoric/ui-kit/blob/main/MAINTAINERS.md) includes an explicit prepack step as well. Just took this back out though so we can be sure if it's needed, but worst case we'll have to do another PR to add it back.